velox/docs/functions/spark/string.rst

```diff
@@ -38,6 +38,32 @@ Unless specified otherwise, all functions return NULL if at least one of the arg
 
     Returns the length of ``string`` in characters.
 
+.. spark:function:: ltrim(srcStr) -> varchar
+
+    Removes leading 0x20(space) characters from ``srcStr``. ::
+
+        SELECT ltrim('  data  '); -- "data  "
+
+.. spark:function:: ltrim(trimStr, srcStr) -> varchar
```

Comment:
perhaps, trimStr, srcStr -> trimCharacters, string for consistency

Is it allowed to have duplicate characters specified in trimStr? Can trimStr be empty?